### PR TITLE
Website: post update revisions

### PR DESCRIPTION
**Pull request by jasoncostello**
Ahoy 👋 

This PR follows up on the recent website design update. The changes are:

- Optical type adjustments
- Increase body copy and heading contrast
- Remove hero graphic (for now)
- Correct the logo color
- Use brand compliant, full color logos for integrations
- Update shadow styles on buttons

It looks like: 

screencapture-localhost-4567-1491935952089

The docs remain the same except for the subtle increase in body copy contrast. 

cc @captainill @sethvargo

### CODE REVIEW

File: website/source/index.html.erb [hunk 1]
```diff
@@ -17,9 +17,6 @@ description: |-
         <a class="button primary" href="/intro/index.html">Get Started</a>
         <a class="button" href="/downloads.html">Download <%= latest_version %></a>
       </div>
-      <div class="col-md-6">
-        <%= inline_svg "hero-image.svg" %>
```
Comment (sethvargo) [inline website/source/index.html.erb]: Did we delete this from the assets/images folder?

Comment (jasoncostello) [inline website/source/index.html.erb]: I deleted the image and resolved the conflict

**Comment by sethvargo** [comment]
👍

**Comment by mwhooker** [comment]
@sethvargo is this ready to go?

**Comment by mwhooker** [comment]
cherry-picked to stable-website and deployed

**Comment by jasoncostello** [comment]
@mwhooker @sethvargo Thanks!

**Pull request merged**